r122/u_inref/NULLCOLLtest.csh

```diff
@@ -0,0 +1,61 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+if ($terminalNoKill == 1) then
+	$gtm_tst/$tst/u_inref/ydb210_dbcreate.csh
+else
+	(expect -d $gtm_tst/$tst/u_inref/ydb210_dbcreate.exp > expect_NULLCOLL.out) >& expect_NULLCOLL.dbg
+	if ($status) then
+		echo "EXPECT-E-FAIL : expect returned non-zero exit status"
+	endif
+
+	mv expect_NULLCOLL.out expect_NULLCOLL.outx	# move .out to .outx to avoid -E- from being caught by test framework
+	# The output is variable on slow vs fast systems and so filter out just the essential part of it to keep it deterministic.
+	perl $gtm_tst/com/expectsanitize.pl expect_NULLCOLL.outx > expect_NULLCOLL_sanitized.outx
+endif
+
+setenv start_time `cat start_time` # start_time is used in naming conventions
+setenv srcLog "SRC_$start_time.log"
+setenv portno `$sec_shell "$sec_getenv ; source $gtm_tst/com/portno_acquire.csh"`
+
+echo "# Shut down source server and set regions to different NULL Collation" >> NULLCOLL.logx
```

Comment:
Why do we need to use NULLCOLL.logx. I would prefer .log instead of .logx anytime we do not expect an error. For the few occasions where an error is expected, we can redirect output to .logx and filter the expected error out (using com/check_error_exist.csh) and append that remaining output back to the .log file that way we build a .log file (instead of a .logx file) throughout this script and other such scripts.